pkgs/applications/video/avidemux/default.nix

```diff
@@ -57,7 +57,12 @@ stdenv.mkDerivation rec {
 
   buildCommand = let
     wrapWith = makeWrapper: filename:
-      "${makeWrapper} ${filename} --set ADM_ROOT_DIR $out --prefix LD_LIBRARY_PATH : ${libXext}/lib";
+      ''
+        ${makeWrapper} ${filename} \
+          --set ADM_ROOT_DIR $out \
+          --prefix LD_LIBRARY_PATH : ${libXext}/lib \
+          --prefix LD_LIBRARY_PATH : $out/lib
```

Comment:
should probably be `--prefix LD_LIBRBARY_PATH : ${lib.makeLibraryPath [ libXext (placeholder "out") ]}` or something like this.

Reply:
Done!